From 47a111a45a541316d7ff9d844362b901f24f41e1 Mon Sep 17 00:00:00 2001
From: Brent Cook <busterb@gmail.com>
Date: Wed, 31 Dec 2014 08:47:45 -0600
Subject: [PATCH 12/12] don't be too stingy on braces

Match if bracing of the block below it to improve readability.
---
 src/usr.sbin/ntpd/client.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/usr.sbin/ntpd/client.c b/src/usr.sbin/ntpd/client.c
index 0271068..4bfbf90 100644
--- a/src/usr.sbin/ntpd/client.c
+++ b/src/usr.sbin/ntpd/client.c
@@ -139,7 +139,7 @@ client_query(struct ntp_peer *p)
 		struct sockaddr *sa = (struct sockaddr *)&p->addr->ss;
 
 		p->query->fd = socket(p->addr->ss.ss_family, SOCK_DGRAM, 0);
-		if (p->query->fd == -1)
+		if (p->query->fd == -1) {
 			if (errno == EAFNOSUPPORT) {
 				log_warn("client_query socket");
 				client_nextaddr(p);
@@ -147,6 +147,7 @@ client_query(struct ntp_peer *p)
 				return (-1);
 			} else
 				fatal("client_query socket");
+		}
 
 #ifdef SO_RTABLE
 		if (p->rtable != -1 &&
-- 
1.9.1

